core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -1180,7 +1180,7 @@ private List<KafkaConnectorRequestModel> getConnectorRequestModels(
           RequestStatus.of(connectorRequest.getRequestStatus()));
       kafkaConnectorRequestModel.setRequestOperationType(
           RequestOperationType.of(connectorRequest.getRequestOperationType()));
-      kafkaConnectorRequestModel.setTeamName(
+      kafkaConnectorRequestModel.setTeamname(
           manageDatabase.getTeamNameFromTeamId(tenantId, kafkaConnectorRequestModel.getTeamId()));
```

Comment:
is the switch from setTeamName to setTeamname on purpose?

Reply:
By introducing BaseRequestModel, reusing the teamname attribute.